pkg/guacrest/openapi.yaml

```diff
@@ -0,0 +1,118 @@
+#
+#  Copyright 2024 The GUAC Authors.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+openapi: "3.0.0"
```

Comment:
hmm , should we use the latest version 3.1.0?

Reply:
oapi-codegen doesn't yet support the features added in 3.1. (See https://github.com/deepmap/oapi-codegen/issues/373).